index.bs

```diff
@@ -1202,124 +1207,125 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |issuedRequests| be a new [=ordered set=].
 
-1. Let |authenticator| be a platform-specific handle whose value identifies an [=authenticator=].
+1. Let |savedCredentialIds| be a new [=map=].
+
+1. Let |authenticators| represent a [=set=] of platform-specific handles, where each value identifies an [=authenticator=]
+    presently available on this platform at a given instant.
+
+    Note: What qualifies an [=authenticator=] as "available" is intentionally unspecified; this is meant to represent how
+    [=authenticators=] can be hot-plugged into (e.g., via USB) or discovered (e.g., via NFC or Bluetooth) by the [=client=] by
+    various mechanisms, or permanently built into the [=client=].
 
 1. Start |lifetimeTimer|.
 
-1. [=set/For each=] |authenticator| that becomes available on this platform during the lifetime of
-    |lifetimeTimer|, perform the following steps:
+1. [=While=] |lifetimeTimer| has not expired, perform the following actions depending upon |lifetimeTimer|,
+    and the state and response [=set/for each=] |authenticator| in |authenticators|:
 
-    Issue: The definitions of "lifetime of" and "becomes available" are intended to represent how
-    devices are hot-plugged into (USB) or discovered by (NFC) browsers, and are underspecified.
-    Resolving this with good definitions or some other means will be addressed by resolving
-    [Issue #613](https://github.com/w3c/webauthn/issues/613).
+    <dl class="switch">
 
-    1. If <code>|options|.{{PublicKeyCredentialRequestOptions/userVerification}}</code> is set to
-        {{UserVerificationRequirement/required}} and the |authenticator| is not capable of performing [=user verification=],
-        [=iteration/continue=].
+        :   If |lifetimeTimer| expires,
+        ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on
+            |authenticator| and [=set/remove=] |authenticator| from |issuedRequests|.
 
-    1. Let |userVerification| be the <dfn>effective user verification requirement for assertion</dfn>, a Boolean value, as
-        follows. If <code>|options|.{{PublicKeyCredentialRequestOptions/userVerification}}</code>
+        :   If the {{CredentialRequestOptions/signal}} member is [=present=] and the [=AbortSignal/aborted flag=] is set to
+            [TRUE],
+        ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
+            and [=set/remove=] |authenticator| from |issuedRequests|. Then
+            return a {{DOMException}} whose name is "{{AbortError}}" and terminate this algorithm.
 
-            <dl class="switch">
+        :   If an |authenticator| becomes available on this platform,
+        ::  Note: This includes if the |authenticator| was initially available when |lifetimeTimer| started.
```

Comment:
suggest for the Note: 
This includes the case where an |authenticator| was available upon |lifetimeTimer| initiation.